{"name": "Test to verify error correction on repeated task", "status": "passed", "description": "This test case verify that if error correction is applied on repeated task", "parameters": [{"name": "job_execution_by", "value": "'user'"}, {"name": "users_list", "value": "['process.publisher.03', 'facility.admin.03']"}], "start": 1733764078570, "stop": 1733764097382, "uuid": "40c496d2-7f72-46ca-a1aa-d009911d2cce", "historyId": "54bdc446f356af20e4abbe80d0c4c5fc", "testCaseId": "261b46a14ddbd310066580bd1c218a33", "fullName": "tests.scenarios.job.test_repeat_task.TestRepeatTask#test_error_correction_on_repeated_task", "labels": [{"name": "tag", "value": "error_correction_on_repeated_task"}, {"name": "tag", "value": "repeat_task"}, {"name": "tag", "value": "job"}, {"name": "tag", "value": "scenario"}, {"name": "parentSuite", "value": "tests.scenarios.job"}, {"name": "suite", "value": "test_repeat_task"}, {"name": "subSuite", "value": "TestRepeatTask"}, {"name": "host", "value": "ubuntu"}, {"name": "thread", "value": "53959-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.scenarios.job.test_repeat_task"}], "links": [{"type": "link", "url": "https://app.clickup.com/t/86cvy0m6j", "name": "https://app.clickup.com/t/86cvy0m6j"}]}